src/frontend/javascript/event-tracking.js

```diff
@@ -0,0 +1,96 @@
+function EventTracking () {
+  this.$bodyElement = document.querySelector('body')
+  this.$mainHeading = this.$bodyElement.querySelector('h1')
+  this.errorStatusCode,
+  this.isErrorPage = this.$mainHeading ? 
```

Comment:
We should extract this to a method. It's a sufficiently complicated expression.

Reply:
@AP-Hunt  extracted into a separate function